ProtocolHandler.pm

```diff
@@ -2,7 +2,7 @@ package Plugins::GoogleMusic::ProtocolHandler;
 
 use strict;
 use warnings;
-use base qw(Slim::Player::Protocols::HTTP);
+use base qw(Slim::Player::Protocols::HTTPS);
```

Comment:
Wrapping this in a conditional to check the version I think should work:
```
if ( Slim::Utils::Versions->compareVersions($::VERSION, '7.9.0') < 0 ) {
    eval "use base qw(Slim::Player::Protocols::HTTP)";
} else {
    eval "use base qw(Slim::Player::Protocols::HTTPS)";
}
```
Only tested on 7.9.1.